tests/ablateLibrary/inputs/chemistry/chemTabTestModel_1/weights.csv

```diff
@@ -0,0 +1,54 @@
+,zmix,CPV_0,CPV_1
```

Comment:
The new ChemTab weights file will not include zmix as a column. This is because we do not project the mass fraction vector to get zmix. I have pushed in the updated weights file as well as the inverse weights file.